app/components/agenda/agendaitem/agendaitem-decision.js

```diff
@@ -77,7 +77,7 @@ export default class AgendaitemDecisionComponent extends Component {
     const defaultAccessLevel = await this.store.findRecordByUri(
       'concept', subcaseIsConfidential
         ? CONSTANTS.ACCESS_LEVELS.VERTROUWELIJK
-        : CONSTANTS.ACCESS_LEVELS.INTERN_REGERING
+        : CONSTANTS.ACCESS_LEVELS.INTERN_OVERHEID
```

Comment:
I assume this is from a different PR that hasn't been merged yet?

Reply:
default of a decision is `INTERN_OVERHEID` from https://kanselarij.atlassian.net/browse/KAS-3826. Is on DEV.
I think this is a problem that the base branch is not up to date with development, but this branch is.
Goes back to my comment "the base of this branch is feature/KAS-3796-result-cards but there is no PR related for that branch that merges into development."

Correct flow should be = update base branch with development > update forked branch with base branch.
It seems we are doing = update forked branch with development which causes this kind of diffs

Reply:
Going to do this now, this PR diff is bothering me